### PR TITLE
chore: note on package versions below v11.0.0

### PR DESCRIPTION
**Pull request by elnyry-sam-k**
first time being upgraded

### CODE REVIEW

File: contributors-guide/standards/versioning.md [hunk 1]
```diff
@@ -9,9 +9,10 @@ This document provides guidelines regarding the versioning strategy used for the
...
-2. At a high-level, we will still follow the vX.Y.Z format, but X represents ‘Major’ version, Y represents ‘Minor’ version and Z represents ‘patch’ version. Minor fixes, patches affect increments to ‘Z’, whereas non-breaking functionality changes affect changes to ‘Y’; breaking changes affect the ‘X’ version.
+2. At a high-level, we will still follow the vX.Y.Z format, but X represents ‘Major’ version, Y represents ‘Minor’ version and Z represents ‘patch’ version. Minor fixes, patches affect increments to ‘Z’, whereas non-breaking functionality changes affect changes to ‘Y; breaking changes affect the ‘X’ version.
 3. Along with these, suffixes such as “-snapshot”, “-patch”, “-hotfix” are used as relevant and on need basis (supported by CI config).
 4. So starting with 11.0.0 (primarily for Helm, but for individual services as well) for PI-11, the proposal is to move to pure [SemVer](https://semver.org/).
+5. This implies that for any new release of a package/service below X=11 will first be baselined to v11.0.0 and from then on follow standard SemVer guidelines as discussed above
```
Comment (lewisdaly) [inline contributors-guide/standards/versioning.md:15]: I'm not sure I agree. Why not just start new services at `v1.0.0` and let them go from there?

Comment (elnyry-sam-k) [reply contributors-guide/standards/versioning.md:15]: I think at the time SemVer was proposed we agreed (as far as I remember :-) ) that `11`  will be our `1`... This is more for services lagging at say v9.xx or v10.x.x or v8.x.x and not new services as such.. So far we adopted this so far, for example, services at 9.x.x were bumped to v11.0.0 to baseline

For brand new services, I think using v1.0.0 etc is ok

Comment (elnyry-sam-k) [reply contributors-guide/standards/versioning.md]: @lewisdaly added an additional comment, please review again.. Thank you!

**Review by lewisdaly — APPROVED** [review]
Looks good. Thanks for allowing me to be pedantic.

**Pull request merged**